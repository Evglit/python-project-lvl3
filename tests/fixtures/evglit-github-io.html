<!DOCTYPE HTML>
<html>
    <head>
        <title>Язык программирования Python</title>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <link type="text/css" href="css/style.css" rel="stylesheet">
    </head>  
    <body>
        <div lign="center">
            <h1 align=center>Язык программирования Python</h1>
            <table>
                <tr>
                    <td>
                        <img src="images/python.png" height="200" align="left" hspace=50 vspace=10 alt="Python" title="Python">
                    </td>
                    <td>
                        <p>Python – это универсальный современный язык программирования высокого уровня, 
                            к преимуществам которого относят высокую производительность программных решений и структурированный, 
                            хорошо читаемый код. Python может поддерживать широкий перечень стилей разработки приложений, в том числе, 
                            очень удобен для работы с ООП и функционального программирования. Язык назван в честь известного комедийного шоу "Летающий Цирк Монти Пайтона"</p>
                        <p><b>Сферы применения языка Python</b></p>
                        <ul>
                            <li>Web-разработка
                            <li>Графический интерфейс
                            <li>Базы данных
                            <li>Системное программирование
                            <li>Сложные вычислительные процессы
                            <li>Машинное обучение
                            <li>Автоматизация процессов
                            <li>Игровая индустрия
                        </ul>
                    </td>
                </tr>
            </table>
            <hr>
            <h2 align=center>Проекты</h2>
            <ol>
                <li><a href="https://github.com/Evglit/python-project-lvl1">Игры разума</a>
                <li><a href="https://github.com/Evglit/python-project-lvl1">Вычислитель отличий</a>
                <li><a href="https://github.com/Evglit/python-project-lvl1">Загрузчик страниц</a>
                <li>Менеджер задач
            </ol>
            <script src="js/runtime.js"></script>
        </div>
    </body>  
</html>