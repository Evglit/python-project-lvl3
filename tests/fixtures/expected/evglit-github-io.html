<!DOCTYPE HTML>
<html>
 <head>
  <title>
   Язык программирования Python
  </title>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
  <link href="evglit-github-io_files/evglit-github-io-css-style.css" rel="stylesheet" type="text/css">
 </head>
 <body>
  <div lign="center">
   <h1 align="center">
    Язык программирования Python
   </h1>
   <table>
    <tr>
     <td>
      <img align="left" alt="Python" height="200" hspace="50" src="evglit-github-io_files/evglit-github-io-images-python.png" title="Python" vspace="10">
     </td>
     <td>
      <p>
       Python &ndash; это универсальный современный язык программирования высокого уровня, 
                            к преимуществам которого относят высокую производительность программных решений и структурированный, 
                            хорошо читаемый код. Python может поддерживать широкий перечень стилей разработки приложений, в том числе, 
                            очень удобен для работы с ООП и функционального программирования. Язык назван в честь известного комедийного шоу "Летающий Цирк Монти Пайтона"
      </p>
      <p>
       <b>
        Сферы применения языка Python
       </b>
      </p>
      <ul>
       <li>
        Web-разработка
        <li>
         Графический интерфейс
         <li>
          Базы данных
          <li>
           Системное программирование
           <li>
            Сложные вычислительные процессы
            <li>
             Машинное обучение
             <li>
              Автоматизация процессов
              <li>
               Игровая индустрия
              </li>
             </li>
            </li>
           </li>
          </li>
         </li>
        </li>
       </li>
      </ul>
     </td>
    </tr>
   </table>
   <hr>
   <h2 align="center">
    Проекты
   </h2>
   <ol>
    <li>
     <a href="https://github.com/Evglit/python-project-lvl1">
      Игры разума
     </a>
     <li>
      <a href="https://github.com/Evglit/python-project-lvl1">
       Вычислитель отличий
      </a>
      <li>
       <a href="https://github.com/Evglit/python-project-lvl1">
        Загрузчик страниц
       </a>
       <li>
        Менеджер задач
       </li>
      </li>
     </li>
    </li>
   </ol>
   <script src="evglit-github-io_files/evglit-github-io-js-runtime.js">
   </script>
  </div>
 </body>
</html>